webapp/src/main/java/com/box/l10n/mojito/metrics/NamingConventionBuilder.java

```diff
@@ -0,0 +1,35 @@
+package com.box.l10n.mojito.metrics;
+
+import com.google.common.base.Strings;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.micrometer.core.lang.Nullable;
+
+public class NamingConventionBuilder {
+
+    public static NamingConvention build(StatsdNamingConventionConfig config) {
+
+        NamingConvention convention = config.getNamingConvention();
+        String namePrefix;
+
+        if (Strings.isNullOrEmpty(config.getNamePrefix()) || config.getNamePrefix().endsWith(".")) {
+            namePrefix = config.getNamePrefix();
+        } else {
+            namePrefix = config.getNamePrefix() + ".";
+        }
+
+        return new NamingConvention() {
+
+            @Override
+            public String name(String name, Meter.Type type, @Nullable String baseUnit) {
+                return namePrefix + convention.name(name, type, baseUnit);
```

Comment:
With `NamingConvention` being defined via configuration, and without any re-direction, we can use the convention directly from here.